vispy/gloo/wrappers.py

```diff
@@ -694,15 +698,35 @@ def get_gl_configuration():
                  depth=(GL_DEPTH, 33302),
                  stencil=(GL_STENCIL, 33303))
     for key, val in sizes.items():
-        config[key + '_size'] = fb_param(gl.GL_FRAMEBUFFER, val[0], val[1])
-    val = fb_param(gl.GL_FRAMEBUFFER, GL_FRONT_LEFT,
-                   GL_FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING)
-    if val not in (gl.GL_LINEAR, GL_SRGB):
-        raise RuntimeError('unknown value for SRGB: %s' % val)
-    config['srgb'] = True if val == GL_SRGB else False  # GL_LINEAR
-    config['stereo'] = True if gl.glGetParameter(GL_STEREO) else False
-    config['double_buffer'] = (True if gl.glGetParameter(GL_DOUBLEBUFFER)
-                               else False)
+        try:
+            param = fb_param(gl.GL_FRAMEBUFFER, val[0], val[1])
+            gl.check_error('post-config check')
+        except RuntimeError as exp:
+            logger.warning('Failed to get size %s: %s' % (key, exp))
+        else:
+            config[key + '_size'] = param
+
+    try:
+        val = fb_param(gl.GL_FRAMEBUFFER, GL_FRONT_LEFT,
+                       GL_FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING)
+        gl.check_error('post-config check')
+    except RuntimeError as exp:
+        logger.warning('Failed to get sRGB: %s' % (exp,))
+    else:
+        if val not in (gl.GL_LINEAR, GL_SRGB):
+            logger.warning('unknown value for SRGB: %s' % val)
+        else:
+            config['srgb'] = (val == GL_SRGB)
+
+    for key, enum in (('stereo', GL_STEREO),
+                      ('double_buffer', GL_DOUBLEBUFFER)):
+        val = gl.glGetParameter(enum)
+        try:
+            gl.check_error('post-config check')
+        except Exception as exp:
+            logger.warning('Failed to get %s: %s' % (key, exp))
+        else:
+            config[key] = bool(val)
```

Comment:
Do we still need these changes? Might be good to add a comment why they're needed

Reply:
Again, this was you. I was hoping you have a better idea why they're needed.

Reply:
No recollection :(

Reply:
I changed the `except Exception` to `except RuntimeError` in an effort to have more explicit exception handling. I don't have a huge issue with more explicit warnings and error messages. We'll see if they still pass.